src/watchful/client.py

```diff
@@ -1081,6 +1081,22 @@ def export_preview(mode: str = "ftc") -> Optional[Dict]:
     return api("export_preview", mode=mode)
 
 
+def export_project() -> http.client.HTTPResponse:
+    """
+    This function returns a consolidated version (a single *.hints file) of the
+    currently open project.
+
+    :return:  The HTTP response from the connection request.
```

Comment:
I think we specify GET endpoints to return a summary. If there is specificity to the expected `return`ed HTTP response for this endpoint, then it will be good to add that specificity here too.

Reply:
Added more comment to specify what is being returned here.